game/src/Credits/Credits.gd

```diff
@@ -0,0 +1,98 @@
+extends Control
+
+signal backButtonPressed
+
+###############
+# Credits CSV format
+# The project title row is the only requirement within the csv file, however
+# it can be on any row, so long as it exists.
+# ----------------------
+# title,project-title
+# role-name,person-name
+# role-name,person-name
+# role-name,person-name
+# ...
+###############
+
+@export var coreCreditsPath : String = 'res://common/credits.csv'
+
+# REQUIREMENTS:
+# * 1.5 Credits Menu
+# * SS-17
+
+# REQUIREMENTS
+# * FS-4
+func loadCreditFile(path):
+	var roles = {}
+	var coreCredits = FileAccess.open(path, FileAccess.READ)
+	
+	while not coreCredits.eof_reached():
+		var line = coreCredits.get_csv_line()
+		
+		# If the line does not have an identifiable role or is empty then skip it
+		if line[0] == '':
+			continue
+		
+		var role: String = line[0].to_upper()
+		var person: String = line[1]
+		
+		if role not in roles.keys():
+			roles[role] = []
+			roles[role].push_back(person)
+		else:
+			roles[role].push_back(person)
+			
+	return roles
+
+# REQUIREMENTS:
+# * UI-34, UI-35
+func drawProjectCredits(project):
+	var projectParent = VBoxContainer.new()
+	var projectLabel = Label.new()
+	
+	projectParent.name = 'Credits-' + project['TITLE'][0]
+	projectLabel.name = 'Label-' + project['TITLE'][0]
+	projectLabel.text = project['TITLE'][0]
+	projectLabel.horizontal_alignment = HORIZONTAL_ALIGNMENT_CENTER
+	projectLabel.label_settings = load("res://src/Credits/ProjectLabelSettings.tres")
```

Comment:
Construction should not be performed here in GDScript? 

This only requires `projectLabel.text = project['TITLE'][0]` but even for that I would encourage separating the title from the csv, for now leaving it in the scene anyway. The rest of this should be part of the Credits scene. In the future if we're gonna put the modders in this list, a csv would only list the role and name, the project name will be handled separately.

Reply:
Moders credits was agreed to be a separate CSV file found in the mod directory, hence the need for a title within the CSV to identify it

Reply:
However theoretically that title could be thrown into a some sort of meta data file and handled separately but it wasn't important for now